src/test/components/DragAndDrop.test.js

```diff
@@ -58,5 +58,13 @@ describe('app/DragAndDrop', () => {
 
     fireEvent.dragExit(dragAndDrop);
     expect(overlay.classList).not.toContain('dragging');
+
+    fireEvent.drop(dragAndDrop);
```

Comment:
With a "drop" event, we should have the dropped file information. This is done with the property [dataTransfer](https://developer.mozilla.org/en-US/docs/Web/API/DragEvent/dataTransfer), so this call would look like this:
```js
const files = [ file ];
fireEvent.drop(dragAndDrop, { dataTransfer: { files }});
```

Then the test will wait until everything is loaded.

I think we can reuse part of what's in https://github.com/firefox-devtools/profiler/blob/57954856d9bddbff2ff1af7f9a1cf98427631cbb/src/test/store/receive-profile.test.js#L1173:
* how to mock FileReader and File
* how to test the result